log/logger_setup.go

```diff
@@ -97,7 +116,7 @@ func SetupGlobalLogger() {
 
 	for x := range SubLoggers {
 		SubLoggers[x].Levels = splitLevel(GlobalLogConfig.Level)
-		SubLoggers[x].output = getWriters(&GlobalLogConfig.SubLoggerConfig)
+		SubLoggers[x].output, _ = getWriters(&GlobalLogConfig.SubLoggerConfig)
```

Comment:
How about
```
	var err error
	for x := range SubLoggers {
		SubLoggers[x].Levels = splitLevel(GlobalLogConfig.Level)
		SubLoggers[x].output, err = getWriters(&GlobalLogConfig.SubLoggerConfig)
		if err != nil {
			fmt.Printf("Logger setup error, unable to set sublogger output %v %v\n", SubLoggers[x].name, err)
		}
	}
```
Then at least a user will see:
`Logger setup error, unable to set sublogger output WEBSOCKET unhandled output writer: asdf`